rsmtool/rsmtool.py

```diff
@@ -260,6 +262,16 @@ def run_experiment(config_file_or_obj_or_dict, output_dir, overwrite_output=Fals
     (pred_config, pred_data_container) = modeler.predict_train_and_test(
         train_for_prediction, test_for_prediction, processed_config
     )
+    pred_config_dict = pred_config.to_dict()
+    for key, attr_name in [
+        ("train_predictions_mean", "train_predictions_mean"),
+        ("train_predictions_sd", "train_predictions_sd"),
+        ("human_labels_mean", "h1_mean"),
+        ("human_labels_sd", "h1_sd"),
+    ]:
+        setattr(modeler, attr_name, pred_config_dict[key])
+    with open(join(csvdir, f"{configuration['experiment_id']}.model"), "wb") as model_file:
+        joblib.dump(modeler, model_file)
```

Comment:
I think instead of doing this here, it might be better to add `save()` method to `Modeler` that encapsulates this and then call `modeler.save(model_file)` here.

Reply:
Yeah, I'll do that.